lib/webmock/fixtures/manager.rb

```diff
@@ -3,6 +3,103 @@ module WebMock
   module Fixtures
     # Class used to manage WebMock fixtures
     class Manager
+      # Available fixtures which have been loaded
+      @registered_fixtures = {}
```

Comment:
Ah, nvm. `registered_fixtures` has the method below but `started_fixtues` has none.